sapp/ui/filter_predicates.py

```diff
@@ -6,8 +6,19 @@
 from __future__ import annotations
 
 from abc import ABC, abstractmethod
-from typing import TYPE_CHECKING, Generic, List, Optional, Sequence, Set, TypeVar, Union
-
+from typing import (
+    TYPE_CHECKING,
+    Generic,
+    List,
+    Optional,
+    Pattern,
+    Sequence,
+    Set,
+    TypeVar,
+    Union,
+)
+
+import js_regex
```

Comment:
@abishekvashok Why do we need to import js_regex can't we use the regular python regexes? js_regex https://pypi.org/project/js-regex/ looks like it simply support the javascript regex syntax which is not really something we care about. Moreover this adds a bit of maintenance burden since we would need to support this dependency  internally. Can we use normal python regex instead :)

Reply:
@r0rshark yes we can. But there are differences between the regex engines used by javascript and python. I switched to js-regex following this comment, https://github.com/facebook/sapp/pull/38#discussion_r690458934

But yeah, I will make the change and remove `js-regex` if we don't care much about the numbers in the frontend :)